client/rechallenge/transformers.js

```diff
@@ -26,6 +26,13 @@ const transformersForHtmlJS = {
       transformer: function addLoopProtect(file) {
         return updateContents(loopProtect(file.contents), file);
       }
+    },
+    {
+      name: 'replace-nbsp',
+      transformer: function replaceNBSP(file) {
+        const nbspRegExp = new RegExp(String.fromCharCode(160), 'g');
```

Comment:
Nice catch.

Can you add this as a property of the object instead of creating per function call.